include/veriblock/entities/vbkblock.hpp

```diff
@@ -55,6 +59,30 @@ struct VbkBlock {
    */
   void toVbkEncoding(WriteStream& stream) const;
 
+  /*
+   * TODO
+   */
+  uint32_t getDifficulty() const;
+
+  /*
+   * TODO
+   */
+  uint32_t getBlockTime() const;
```

Comment:
here it is just a getter for `timestamp`

Reply:
Yes but we should keep it documented.